packages/fhir-converter/src/lib/handlebars-converter/handlebars-helpers.js

```diff
@@ -1338,6 +1338,33 @@ module.exports.external = [
       return parseRange(obj);
     },
   },
+  {
+    name: "buildPresentedForm",
+    description: "Builds a presented form array",
+    func: function (b64String, component) {
+      const presentedForm = [];
+      if (b64String) {
+        presentedForm.push({
+          contentType: "text/plain",
```

Comment:
is the content-type always text/plain here?

Reply:
hmm.. well, it's always a b64-encoded string, right? which is a plain text imo. But if this string encodes an html table, do you think we should change the contentType to `text/html`? in your opinion, how important is that distinction?

Reply:
ab64 encoded string could be anything. we get b64 and then its a pdf or a tiff or whatever right? 
In lieu of all the conversations we had about being extra cautious and careful about making changes when it relates to data we present to customers (and this data is definetly that). 

In this case, i=based on the way we created these b64s, is it always a text/html? I think it might be

Reply:
Yah from looking at `cda.js`, I think text/html is the appropriate type here always based on the logic we have

Reply:
sounds good! Thx for the catch!